index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>portfolio</title>
+    <link rel="stylesheet" href="main.css">
+</head>
+<body>
+    <div class="wrapper">
+            <header class="header-section">
+                <nav class="logo"><div>
+                    <a href="#">My Logo</a>
+                </div></nav>
+                <nav><div>
+                    <a href="#"><img src="/images/Enabled.png" alt="enable image"></a>
+                </div></nav>
```

Comment:
- [ ] What the previous reviewer meant for you to do was to have the ` <a href="#">My Logo</a>` and the ` <a href="#"><img src="/images/Enabled.png" alt="enable image"></a>` inside a single `nav` tag. Nav tags are used to __group__ elements that navigate you around the page, it is not the container for a lone element. Same goes for the `a` tag, since both the logo and the image reference the same thing you can have them inside a single `a` tag. If you want containers for styling purposes, it is better to use a `div`. I know the instructions can be confusing sometimes but it's okay, you can always leave a message about your confusion in the conversation or message the reviewer on slack. I'm sure they would be more than happy to help out. You can reference the following code as a guide.

```
<a href="#">
  <div class="logo>My Logo</div>
  <img src="/images/Enabled.png" alt="enable image">
</a>
```